src/common/snippets/src/pass/fq_decomposition.cpp

```diff
@@ -211,8 +213,8 @@ bool ov::snippets::pass::FakeQuantizeDecomposition::getScalesAndShifts(
             float il = input_low[input_low.size() == 1 ? 0 : i];
             float ih = input_high[input_high.size() == 1 ? 0 : i];
 
-            isc[i] = (levels - 1) / (ih - il);
-            ish[i] = -il * isc[i];
+            isc[i] = (z0 - 1) / (ih - il);
```

Comment:
Maybe I missed something but why did you replace `level` with `z0` in formulas for scales `isc` and `osc`? Moreover, for U8 `z0=0`, then these formula is fully incorrect, is not it? Please show me the formula in the [documentation](https://github.com/aobolensk/openvino/blob/93e6d2f80a9835593cd67aab5580b0ae5b0ab162/src/plugins/intel_cpu/docs/fake_quantize.md) where we have to use `z0` instead of `levels` in scale formulas.

From my [judgments](https://github.com/openvinotoolkit/openvino/pull/30176#discussion_r2073027964), we need to add only `z0` to `osh` and `ish`